### ISSUE TITLE
le á n'est pas bon

### ISSUE THREAD

**Issue by Fil** [severo/portfolios-redcartografica#1]
il faut peut-être essayer en majuscules ou changer de police
<img width="386" alt="capture d ecran 2018-03-24 a 09 33 10" src="https://user-images.githubusercontent.com/7001/37862004-63e4c1d2-2f46-11e8-8ecf-547b2acc342b.png">

**Comment by Fil** [comment]
c'est amusant c'est le "text-transform: full-width;" qui transforme toutes les lettres sauf le á, qui est la seule lettre correcte en fait…

**Comment by severo** [comment]
Oui, j'ai fait un truc vite fait. D'ailleurs je ne sais pas si on garde ce nom ni si on l'écrit en espagnol, alors ce n'est pas forcément prioritaire. On peut simplement virer l'accent.
